cms/release_calendar/models.py

```diff
@@ -233,3 +233,10 @@ def active_bundle(self) -> Optional["Bundle"]:
             return None
         bundle: Bundle | None = self.bundles.active().first()  # pylint: disable=no-member
         return bundle
+
+    @property
+    def live_status(self) -> Optional[ReleaseStatus]:
+        if not self.pk:
+            return None
+        live_page = ReleaseCalendarPage.objects.filter(pk=self.pk).live().only("status").first()
```

Comment:
I like this approach.
It is worth adding some dev notes as to why not using https://docs.wagtail.org/en/stable/reference/models.html#wagtail.models.DraftStateMixin.live_revision